File: spec/code_samples/Python_requests/list/post.py
import requests

url = "https://analyzemail.com/api/v1/list"

querystring = {"username":"SOME_STRING_VALUE"}

payload = "{\"filename\":\"<ADD STRING VALUE>\",\"emails\":[\"<ADD STRING VALUE>\"]}"
headers = {'content-type': 'application/json'}

response = requests.request("POST", url, data=payload, headers=headers, params=querystring)

print(response.text)

File: spec/code_samples/Node_request/list/post.js
var request = require("request");

var options = { method: 'POST',
  url: 'https://analyzemail.com/api/v1/list',
  qs: { username: 'SOME_STRING_VALUE' },
  headers: { 'content-type': 'application/json' },
  body:
   { filename: '<ADD STRING VALUE>',
     emails: [ '<ADD STRING VALUE>' ] },
  json: true };

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});
